recipe/bld.bat

```diff
@@ -0,0 +1,11 @@
+setlocal EnableDelayedExpansion
+
+:: Copy the [de]activate scripts to %LIBRARY_PREFIX%\etc\conda\[de]activate.d.
+:: This will allow them to be run on environment activation.
+FOR %%F IN (activate deactivate) DO (
+    IF NOT EXIST %LIBRARY_PREFIX%\etc\conda\%%F.d MKDIR %LIBRARY_PREFIX%\etc\conda\%%F.d
+    COPY %RECIPE_DIR%/%%F.bat %LIBRARY_PREFIX%\etc\conda\%%F.d\keras_%%F.bat
```

Comment:
the proper way to set env vars on windows is like this:

```set "VARNAME=some string"```

weird, but that's the way it works.  Spaces are allowed with that syntax.